webapp/channels/src/components/admin_console/team_channel_settings/convert_and_remove_confirm_modal.tsx

```diff
@@ -62,21 +71,37 @@ const ConvertAndRemoveConfirmModal = ({show, onConfirm, onCancel, displayName, t
     );
 
     const message = (
-        <div>
+        <>
             <p>
-                <FormattedMarkdownMessage
+                <FormattedMessage
                     {...convertMessage}
-                    values={{displayName}}
+                    values={{
+                        displayName: <strong>{displayName}</strong>,
+                    }}
                 />
             </p>
             <p>
                 <FormattedMessage
-                    id='admin.team_channel_settings.removeConfirmModal.messageChannel'
-                    defaultMessage='{amount, number} {amount, plural, one {user} other {users}} will be removed. They are not in groups linked to this channel. Are you sure you wish to remove these users?'
+                    {...convertMessageSecondLine}
+                    values={{
+                        displayName: <strong>{displayName}</strong>,
+                    }}
+                />
+            </p>
+            <p>
+                <FormattedMessage
```

Comment:
It's my understanding the translation process always want to keep these kind of sentences unified in a single line to preserve context. Splitting it might potentially make the translators work harder, but I'd like to read @cwarnermm opinion on this one.

Reply:
If I'm understanding context correctly, the split is between "x will be removed" and "are you sure you want to do this?"

If there's precedent across the product for these types of string splits, then I'm in favour of doing so here as well.

Reply:
There are other places where we have used these types of split as well. Since its a better to have `p` for each line this is much better than using `br`.